scripts/gceworker.sh

```diff
@@ -19,6 +19,17 @@ fi
 
 function start_and_wait() {
     gcloud compute instances start "${1}"
+    if [ -z "${GCEWORKER_NO_FIREWALL_WARNING-}" ]; then
+	      cat <<EOF
+	      As of July 2024, due to an SSH vulnerability, access to gceworkers is permitted only from home and office IPs[1].
```

Comment:
What if you printed the warning only if SSH fails? At this point, most people who regularly use GCEWorkers must have the firewall rules set up. If you do it so that the warning only happens if the SSH fails, then you avoid spamming those people who are already on top of it.

Reply:
Not a bad thought - but many work from home and from the office, but we only get one firewall rule. Don't want to spend my cycles on diminishing returns.